extensions/pyRevitTools.extension/pyRevit.tab/Analysis.panel/Tools.stack/Analyse.pulldown/Get In Place Family Count.pushbutton/script.py

```diff
@@ -0,0 +1,42 @@
+"""
+Get a total count and types of In Place Family in the current document.
+Copyright (c) 2019 Jean-Marc Couffin
+https://github.com/jmcouffin
+--------------------------------------------------------
+PyRevit Notice:
+Copyright (c) 2014-2019 Ehsan Iran-Nejad
+pyRevit: repository at https://github.com/eirannejad/pyRevit
+"""
+
+__title__ = 'Get In Place Family count'
```

Comment:
If the bundle name is identical to `__title__` you don't need to set this. pyRevit uses the bundle name as default.